doc/manpages/man1/nbp.1.md

```diff
@@ -22,6 +22,11 @@ registered on the AppleTalk internet. *nbpname* is parsed by
 and an \`*\**' for *zone* means the local zone. The default values are
 taken from the NBPLKUP environment variable, parsed as an *nbpname*.
 
+If -A is specified, this address is used as the local address to be used for the lookup.
+If -D is specified, this address is used as the destination address to send the lookup to.
+The -f and -l options allow FwdReq and LkUp operations to be used instead of the
+default BrRq.
```

Comment:
Might be worth adding a sentence on what distinguishes these three operations. I have to be honest that I don't know myself off the top of my head. (Unless I take the time to read up on it.)